generate-changelog.py

```diff
@@ -11,13 +11,20 @@
 from jira import JIRA, JIRAError
 from datetime import datetime
 
-# point to your jira installation
-jira = JIRA(server=('https://jira.yourdomain.com'), auth=('changelogbot', 'cryp71cp455w0rd'))
+# The base URL of your Jira installation.
+# Must not have a trailing slash!
+jiraBaseUrl = 'https://jira.yourdomain.com'
+
+jira = JIRA(server=(jiraBaseUrl), auth=('changelogbot', 'cryp71cp455w0rd'))
 # configure your jira project or just leave it to find all
 project_format = '[A-Z][A-Z]+'
 # define jira project to create version
 project_version = 'CORE'
 
+# Flag whether the script should automatically add the "version" information
+# to the issues in Jira.
+is_update_fix_versions_enabled = True
```

Comment:
for me thats the main feature :D will not take it, we can think about a dry run though if more people need that.